[2021-09-27 14:50:03,204] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T14:40:00+00:00 [queued]>
[2021-09-27 14:50:03,223] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T14:40:00+00:00 [queued]>
[2021-09-27 14:50:03,225] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:50:03,226] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:50:03,227] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:50:03,238] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T14:40:00+00:00
[2021-09-27 14:50:03,242] {standard_task_runner.py:52} INFO - Started process 13782 to run task
[2021-09-27 14:50:03,246] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T14:40:00+00:00', '--job-id', '65', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmplnkjj69t', '--error-file', '/tmp/tmpw3ulqrkx']
[2021-09-27 14:50:03,250] {standard_task_runner.py:77} INFO - Job 65: Subtask say_hello
[2021-09-27 14:50:03,306] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T14:40:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-27 14:50:03,365] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T14:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T14:40:00+00:00
[2021-09-27 14:50:03,368] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-27 14:50:03,369] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 14:50:03,384] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T144000, start_date=20210927T145003, end_date=20210927T145003
[2021-09-27 14:50:03,420] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:50:03,470] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
